hooks/playbooks/nova_wait_for_compute_service.yml

```diff
@@ -0,0 +1,47 @@
+---
+- name: Run compute service list until exptected compute count present
+  hosts: "{{ cifmw_target_hook_host | default('localhost') }}"
+  gather_facts: false
+  vars:
+    _number_of_computes: 0
+    _retries: 25
+    _cell_conductor: null
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+    PATH: "{{ cifmw_path }}"
+  tasks:
+    - name: Wait for expected compute services
+      cifmw.general.ci_script:
+        output_dir: "{{ cifmw_basedir }}/artifacts"
+        executable: "/bin/bash"
+        script: |
+          set -xe -o pipefail
+          COMPUTES={{ _number_of_computes }}
+          RETRIES={{ _retries }}
+          COUNTER=0
+          oc project {{ namespace }}
+          until [ $(oc rsh openstackclient openstack compute service list --service nova-compute -f value | wc -l) -eq "$COMPUTES" ]; do
+            if [[ "$COUNTER" -ge "$RETRIES" ]]; then
+              exit 1
+            fi
+            COUNTER=$[$COUNTER +1]
+            sleep 10
+          done
+    - name: Run nova-manage discover_hosts and wait for host records
+      cifmw.general.ci_script:
+        output_dir: "{{ cifmw_basedir }}/artifacts"
+        executable: "/bin/bash"
+        script: |
+          set -xe -o pipefail
+          COMPUTES={{ _number_of_computes + 4 }}
```

Comment:
why +4 i dont think this is correct

Reply:
The ouput is a table with 4 lines in addition to the lines 
```
+------------------+
| <headers>        |
+------------------+
| <content>        |
+------------------+
```